src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -151,6 +154,93 @@
     </ItemGroup>
   </Target>
 
+  <!-- #################################### -->
+  <!-- Generate BuildVersion.props -->
+  <!-- #################################### -->
+  <Target Name="CreateOrUpdateCurrentVersionFile" Condition="'$(SkipVersionGeneration)'!='true'" DependsOnTargets="GenerateCurrentVersion">
+    <!-- Error out if GeneratedBuildNumberMajor is empty since GenerateCurrentVersion should have calculated it already. -->
+    <Error Condition="'$(GeneratedBuildNumberMajor)'==''" Text="Error when generating current version." />
+
+    <PropertyGroup>
+      <VersionSeedSourceComment Condition="'$(VersionSeedSourceComment)'==''">VersionSeedDate was produced by last git commit timestamp.</VersionSeedSourceComment>
```

Comment:
I would reword this a little. "VersionSeedData was produced by taking the timestamp of the last git commit."